client/src/store/types.ts

```diff
@@ -1,9 +1,15 @@
 import {Item} from "./item/types";
+import {User} from "./users/types";
 
 export type RootState = {
   item: ItemState;
 };
 
 export interface ItemState {
   items: Item[];
+}
+export interface UserState {
+  users: User[];
```

Comment:
I see @jarhun88 started this discussion about why we have an array of Users. I agree, this should only be one user. You don't need to make a User type at all at this point.

You mentioned that our app will support more than one user. That is correct, but that is not the responsibility of redux or "states" in general. The (redux) state of the application is supposed to only tell us what the state is for that particular web browser, it is not to represent the entire landscape of the app (in this case, all the users of our application).

Imagine if we had 100k users, do we want to store 100k users on the client side? Is that necessary?

We will solve the problem of managing multiple users once we have a database set up and we utilize third party logins to identify users.